NETProvider/src/FirebirdSql.Data.FirebirdClient/Client/Native/IFbClient.cs

```diff
@@ -269,4 +351,98 @@ public interface IFbClient
 
 		#endregion
 	}
+
+	internal static class IFbClientEx
+	{
+		public static IntPtr isc_dsql_allocate_statement_ex(
```

Comment:
Nope, it won't resolve this. The code looks like 
```client.isc_dsql_allocate_statement(...)``` what function should it call?
I'd like to inject fbclient property in generated stub, but I have problems with manual emitting IL code.